cmd/kubeadm/app/cmd/init.go

```diff
@@ -469,8 +469,10 @@ func runInit(i *initData, out io.Writer) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to create client")
 	}
+
 	// TODO: NewControlPlaneWaiter should be converted to private after the self-hosting phase is removed.
-	waiter, err := phases.NewControlPlaneWaiter(i.dryRun, client, i.outputWriter)
+	timeout := i.cfg.ClusterConfiguration.APIServer.TimeoutForControlPlane.Duration
```

Comment:
Why declaring this var, which is used only once?

Reply:
That's because the expression to take the timeout value is huge and best viewed on a single line. Also the call to `NewControlPlaneWaiter` on the following line continues to fit on screen.